src/pages/Users/Admins/AddNewAdmin.tsx

```diff
@@ -163,13 +163,20 @@ const AddNewAdmin: React.FC<IAddNewAdmin> = ({ setOpen, open }) => {
         adminSchema,
         additionalValues
       );
-      result.roles = result.roles.map((role: any) => {
-        return {
-          id: role.id,
-          from: role.from,
-          to: role.to,
-        };
-      });
+      if (
+        !result?.roles ||
+        !result?.roles?.find((role: any) => role.id === "ROLE_ADMIN")
```

Comment:
Extract this into a separate variable like `isRoleAdmin` or something similar 
And for the string `ROLE_ADMIN`, use from constants.ts